instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -27,6 +27,7 @@ type gRPCContext struct {
 	messagesReceived int64
 	messagesSent     int64
 	metricAttrs      []attribute.KeyValue
+	record           bool
```

Comment:
Can you add tests for changes in to this file in `instrumentation/google.golang.org/grpc/otelgrpc/test`.  Probably in instrumentation/google.golang.org/grpc/otelgrpc/test/grpc_stats_handler_test.go.

Reply:
I added tests for this file with https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5196/commits/d22a1c98f14feba9d7523142c7dc74e22d590c86 .

```bash
$ cd instrumentation/google.golang.org/grpc/otelgrpc/test \
&& go test -race -covermode=atomic -coverprofile=coverage.out -coverpkg=go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc/... \
&& go tool cover -html=coverage.out -o coverage.html \
&& cat coverage.html | grep "go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go"
...
				<option value="file6">go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go (100.0%)</option>
```